### PR TITLE
init commit

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

File: js/View2D.js [hunk 1]
```diff
@@ -1,426 +1,434 @@
...
+        });
+        this._renderer.setClearColor(0x000000, 0.0);
+        this._width = 0;
+        this._height = 0;
+        this._scale = 1.0;
+        this._mouseAction = null;
+        this._offset = {x: 0, y: 0};
+        this._pixelRatio = 1;
+        this._scene = workspace.scene2d;
+        this._spotLabel = new SpotLabel2D(this);
```
Comment (iprotsyuk) [inline js/View2D.js:26]: does this line cause the dependency on SpotLabel2D as well?

File: js/ViewGroup3D.js [hunk 2]
```diff
@@ -1,162 +1,171 @@
...
+        this._animationFrameRequested = false;
+
+        this._scene = workspace.scene3d;
+        this._scene.addEventListener(Scene3D.Events.CHANGE, this.requestAnimationFrame.bind(this));
+
+        this._div.addEventListener('mousedown', this._onMouseDown.bind(this));
+
+        var divs = this._div.querySelectorAll('.View3d');
+        for (var i = 0; i < divs.length; i++) {
+            this._views.push(new View3D(this, divs[i]));
```
Comment (iprotsyuk) [inline js/ViewGroup3D.js:39]: It seems there should be a dependency on View3D

**Comment by iprotsyuk** [comment]
plz check my comments to the source

**Comment by iprotsyuk** [comment]
:+1:

**Comment by ElDeveloper** [comment]
👍 looks good, ready for merge.

**Pull request merged**